packages/dataviews/src/types.ts

```diff
@@ -17,6 +15,8 @@ interface filterByConfig {
 
 type Operator = 'is' | 'isNot' | 'isAny' | 'isNone' | 'isAll' | 'isNotAll';
 
+export type Item = Record< string, any >;
```

Comment:
I'm quite ignorant of what this code actually does, but I'd assume that `Item` is the type of thing in a _view_.

I'm not requesting a change, but I'm trying to anticipate where this might go. It likely makes sense to relate them in some way so that when working with them, you can express things like "a view of users".

```ts
interface View< Item = Record< string, any > > {
  data: ReadonlyArray< Item >;
}

const userView: View< User > = /* … */;
```

Reply:
Yes, that code looks good. We'd probably benefit from generics here down the road.